documentation/reference/releases/2024-sept-3.0.0.md

```diff
@@ -0,0 +1,45 @@
+  * All existing Namespaces will be retained but will now be referred to as Gateways.
+  * The only remaining trace of Namespace is the `ns.` prefix required in tags for Gateway Service configuration (`tags: [ ns.<gatewayId> ]`).
+* Gateways now feature a customizable display name.
+  * This display name is prominently displayed throughout the API Services Portal.
+  * For existing Namespaces, the default display name will be `<IDIR>'s Gateway`.
+* The Namespace `name` attribute has been renamed to `gatewayId` for Gateways.
+  * This change aims to prevent confusion between `gatewayId` and the new `displayName` attribute for Gateways.
+  * The gatewayId is still used for setting the current Gateway in the CLI (`gwa config set gateway <gatewayId>`) and in tags within Gateway Service configuration (`tags: [ ns.<gatewayId> ]`).
+* Gateway selection user interface has changed.
+  * A new **My Gateways** page has been added, featuring a searchable list of Gateways.
+  * The Gateway dropdown selector has been relocated and updated. It now appears only on relevant API provider pages.
+* New starting page for new users after API provider login.
+  * Users without any Gateways are directed to <https://api.gov.bc.ca/manager/gateways/get-started>.
+  * This page provides a basic introduction and instructions for the new Quick Start tutorial for API providers.
+  * The new Quick Start tutorial offers a simpler option to begin with before progressing to the [Protect an API with Client Credential Flow](https://developer.gov.bc.ca/docs/default/component/aps-infra-platform-docs/tutorials/protect-client-cred/) tutorial (previously called Quick Start).
+  * The revised Quick Start tutorial utilizes a new `quick-start` configuration template.
+* Additional Changes:
+  * Added breadcrumbs on Gateway management pages.
+  * **Gateways** is no longer visible in the navigation bar for non-IDIR users (BCeID, BC Services Card, GitHub).
+  * The **Namespace Access** page is renamed to **Administration Access**.
+
+## GWA-CLI V3
+
+Gateway Command Line Interface Version 3 (GWA-CLI V3.0.1) is a new and enhanced tool for configuring your API Gateway settings. Key improvements include:
+
+* Capability to create a gateway directly from the command line.
+* New default behaviour (`gwa gateway create`) is to autogenerate a unique `gatewayId` and prompt for a custom display name.
+* The `--gateway-id` and `--display-name` flags can be used to specify these attributes.
+* Improvements to error reporting for Gateway Service publication (`gwa apply` and `gwa publish-gateway`). Additional error messaging is shown by default to support troubleshooting.
+* Improvements to Gateway Service configuration creation using templates (`gwa generate-config`). Gateway `service` and `route` names are validated for availability during configuration creation.
+* Templates for implementing common authorization patterns on the API Gateway.
```

Comment:
```suggestion
```
This is not new. There is a new template `quick-start` but it's captured above in Portal.